plugins/robothor_plugin/robothor_environment.py

```diff
@@ -521,3 +521,182 @@ def all_objects_with_properties(
     def visible_objects(self) -> List[Dict[str, Any]]:
         """Return all visible objects."""
         return self.all_objects_with_properties({"visible": True})
+
+
+class RoboThorMultiEnvironment(RoboThorEnvironment):
+    """Wrapper for the robo2thor controller providing additional functionality
+    and bookkeeping.
+
+    See [here](https://ai2thor.allenai.org/robothor/documentation) for comprehensive
+     documentation on RoboTHOR.
+
+    # Attributes
+
+    controller : The AI2THOR controller.
+    config : The AI2THOR controller configuration
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
Cool to see the multi-agent stuff coming here!

What do you think about just merging this directly into the `RoboThorEnvironment`? Since we can just default `agent_id` to be 0 it seems like this would be a nice way to avoid repeating code. I'm not 100% on if `last_event.events` works in the expected way with a single angle but I would hope/assume it does.

Reply:
I just merged it (I confirmed that with one agent we still have a `last_event.events`) 👍